### PR TITLE
feat: Move OPA and Styra DAS to ADOPT

### PR DESCRIPTION
**Pull request by askoriy**

### CODE REVIEW

File: radar/infrastructure_ci_cd/opa.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ blip:
   - date: 2020-03-11
     ring: ASSESS
   - date: 2020-04-13
-    ring: TRIAL
```
Comment (sasjo) [inline radar/infrastructure_ci_cd/opa.yaml:6]: Blips are moved by adding a new entry with a date and ring.
```yaml
  - date: 2020-10-29
    ring: ADOPT
```

File: radar/infrastructure_ci_cd/styra_das.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: Styra DAS
 blip:
   - date: 2020-06-18
-    ring: ASSESS
```
Comment (sasjo) [inline radar/infrastructure_ci_cd/styra_das.yaml:4]: Blips are moved by adding a new entry with a date and ring.
```yaml
  - date: 2020-10-29
    ring: ADOPT
```

File: radar/infrastructure_ci_cd/opa.yaml [hunk 3]
```diff
@@ -3,9 +3,9 @@ blip:
...
+    ring: ADOPT
 description: |
-  Open Policy Agent
+  Open Policy Agent is used in within Hiidentity platform to provide end-user authorization for microservices
```
Comment (sasjo) [inline radar/infrastructure_ci_cd/opa.yaml:8]: We should write `Hii Retail platform` instead of `Hiidentity platform`.

**Review by sasjo — CHANGES_REQUESTED** [review]
The "Things we're assing" section in Styra DAS need to be updated now when the tool has been adopted.

**Comment by sclausson** [comment]
Closing, I want to go another way.

**Pull request closed without merging**